app/components/Inputs/AssetInput/AssetInput.jsx

```diff
@@ -16,17 +20,19 @@ type State = {
 function parsedSymbols(
   symbols = [ASSETS.NEO, ASSETS.GAS, ...Object.keys(TOKENS)]
 ): Array<{ value: string, label: string }> {
-  return symbols.map(symbol => ({
-    value: symbol,
-    label: symbol
-  }))
+  return symbols
+    .map(symbol => ({
+      value: symbol,
+      label: symbol
+    }))
+    .filter(symbol => symbol.label.length > 1)
```

Comment:
To filter out empty string symbols being pulled down from https://github.com/CityOfZion/neo-tokens/blob/master/tokenList.json